projects/fsastorefrontlib/src/cms-components/my-account/address-book/address-form/address-form.component.ts

```diff
@@ -91,6 +91,8 @@ export class FSAddressFormComponent extends AddressFormComponent
 
   verifyAddress(): void {
     this.preventDOMManipulation();
+    this.addressForm.get('firstName').enable();
+    this.addressForm.get('lastName').enable();
     super.verifyAddress();
```

Comment:
Here enabled those two fields to get values of disabled fields.
In the method `super.verifyAddress()` calling `this.addressForm.value` won't get disabled fields from the form.
And `this.preventDOMManipulation()` is protection from DOM misuse in the case of any error during the address verification.

Reply:
What happens if form is not valid, will those fields stay enabled ?

Reply:
Just keep it as is. I not like this idea for enabling fields that should be disabled. Keep them disabled on template.

Reply:
> What happens if form is not valid, will those fields stay enabled ?

Yes, those fileds will stay enabled, but there is no way that user misuse that because of `this.preventDOMManipulation()`  which compare fields value with logged in user.

Reply:
> Just keep it as is. I not like this idea for enabling fields that should be disabled. Keep them disabled on template.

I can do it, but think it's not right way to set disabled state of Reactive forms. If you all agree I'll keep the old code?

Reply:
> Just keep it as is. I not like this idea for enabling fields that should be disabled. Keep them disabled on template.

I would rather not. This is not the way reactive forms should disable fields. Furthermore with previous approach there were browser console warnings for just this reason. We should perhaps discuss it in more detail....

Reply:
> Just keep it as is. I not like this idea for enabling fields that should be disabled. Keep them disabled on template.

I would rather not. This is not the way reactive forms should disable fields. Furthermore with previous approach there were browser console warnings for just this reason. We should perhaps discuss this in more detail....

Reply:
Agreed on daily to add `readonly` property in HTML template instead of `disabled` in model. That way there would be no need for enabling disabled fields, and all fields would submit regularly.